lib/channel-access-token/api/channelAccessTokenClient.ts

```diff
@@ -74,16 +57,32 @@ export class ChannelAccessTokenClient {
     clientAssertionType: string,
     clientAssertion: string,
   ): Promise<ChannelAccessTokenKeyIdsResponse> {
-    const queryParams = {
-      clientAssertionType: clientAssertionType,
-      clientAssertion: clientAssertion,
-    };
+    const requestUrl = new URL("/oauth2/v2.1/tokens/kid", this.config.baseURL);
+
+    const queryParams = new URLSearchParams();
+    queryParams.append("clientAssertionType", String(clientAssertionType));
+    queryParams.append("clientAssertion", String(clientAssertion));
+
+    requestUrl.search = queryParams.toString();
+
+    const resp = await fetch(requestUrl, {
+      method: "GET",
+      headers: {
```

Comment:
`fetch` doesn't throw any error only if http status code is not 2xx. We need to tell error happens to SDK user somehow. 

One of the easiest way is just throwing original exception with http status + error body + header. Also we can define error type from OAS(line-openapi), but few paths defined it...